blueocean-rest-impl/pom.xml

```diff
@@ -76,12 +76,10 @@
         <dependency>
```

Comment:
Check the `jackson-*` deps above ^^^ which you perhaps want to pick up from the Jenkins plugin, but TBD. Ideally you would only be using the plugin dep (after upgrading it).